DueTimer.cpp

```diff
@@ -8,8 +8,6 @@
   Released into the public domain.
 */
 
-#include <Arduino.h>
-#if defined(_SAM3XA_)
```

Comment:
No problem, the chip we use is in the same SAM3XA family. I just changed __arm__, which is too generic, and left the test only in the header file (no need to duplicate this in the source file).